packages/docs/docs/docs/deploying.md

```diff
@@ -26,49 +24,55 @@ Use `npm` to create a `package.json` file:
 npm init
 ```
 
-This command will ask you for details about the project. For this very basic
-guide you can just press enter to accept the default values of each field.
+This command will ask you for details about the project. For this getting started project, just press **Enter** to accept the default values for each field.
```

Comment:
We can replace this by just suggesting `npm init -y`

Reply:
Changed to `npm init -y` and removed the subsequent paragraph, which is no longer needed.